test/judges/test-dimensions-of-terrain.rb

```diff
@@ -150,6 +150,127 @@ def test_total_repositories
     end
   end
 
+  def test_total_releases
+    WebMock.disable_net_connect!
+    stub_github(
+      'https://api.github.com/repos/foo/foo',
+      body: {
+        name: 'foo',
+        full_name: 'foo/foo',
+        private: false,
+        created_at: Time.parse('2024-07-11 20:35:25 UTC'),
+        updated_at: Time.parse('2024-09-23 07:23:36 UTC'),
+        pushed_at: Time.parse('2024-09-23 20:22:51 UTC'),
+        size: 19_366,
+        stargazers_count: 1,
+        forks: 1,
+        default_branch: 'master'
+      }
+    )
+    stub_github(
+      'https://api.github.com/repos/foo/foo/releases?per_page=100',
+      body: [
+        { id: 50, tag_name: '0.0.9', draft: false, published_at: Time.parse('2024-08-22 21:00:00 UTC') },
+        { id: 44, tag_name: '0.0.8', draft: false, published_at: Time.parse('2024-08-19 21:00:00 UTC') },
+        { id: 32, tag_name: '0.0.7', draft: false, published_at: Time.parse('2024-08-15 21:00:00 UTC') },
+        { id: 25, tag_name: '0.0.6', draft: false, published_at: Time.parse('2024-08-14 21:00:00 UTC') },
+        { id: 18, tag_name: '0.0.5', draft: false, published_at: Time.parse('2024-08-12 21:00:00 UTC') },
+        { id: 12, tag_name: '0.0.4', draft: true, published_at: Time.parse('2024-08-10 21:00:00 UTC') },
+        { id: 5, tag_name: '0.0.3', draft: false, published_at: nil },
+        { id: 3, tag_name: '0.0.2', draft: false, published_at: Time.parse('2024-08-03 21:00:00 UTC') },
+        { id: 1, tag_name: '0.0.1', draft: false, published_at: Time.parse('2024-07-25 21:00:00 UTC') }
+      ]
+    )
+    stub_github(
+      'https://api.github.com/repos/foo/foo/git/trees/master?recursive=true',
```

Comment:
@Yegorov can we get an empty repo in this `judge`?
if we have an empty repo, we will get a 404 error

Reply:
@Suban05 And why did you decide that the repository can be unavailable here?
If the repository is not available, we will get a error earlier, for example [here](https://github.com/zerocracy/judges-action/pull/379/files#diff-dd8d8bbb280c2e0463bb5c5cf09197fbb2a0faf43e2baba71f3285c234ecd8e9R155-R169)
Or even [here](https://github.com/zerocracy/fbe/blob/ab6f7e05592acf02b05374796561c6e1ad2afb65/lib/fbe/unmask_repos.rb#L56)

Reply:
@Yegorov I mean the repository without commits. Can we get them in the `dimensions-of-terrain`?

Reply:
@Yegorov I understand. but it's a general question, probably, not about this ticket.
can we get an error in the `dimensions-of-terrain`, if we have this case:
https://api.github.com/repos/Yegorov/empty-repo/git/trees/master?recursive=true
?

I think, we will get an error, if we add this stub:

```rb
    stub_github(
      'https://api.github.com/repos/foo/foo/git/trees/master?recursive=true',
      body: { message: "Git Repository is empty." }, status: 404
    )
```    
and I'm asking if we can get empty repositories in the `dimensions-of-terrain` at all. do we need to consider this case.

Reply:
@Suban05 Yes, you are right, this case I think we need to consider in [here](https://github.com/zerocracy/judges-action/blob/7afc6cd80ad0b0827072292644833b71ff4a7b93/judges/dimensions-of-terrain/dimensions-of-terrain.rb#L91) when count total number of files for all repos.
On the other hand, when counting stars or forks, it doesn’t matter to us whether the repository is empty or not

Reply:
@yegor256 can we process empty repositories (without commits) in the `dimensions-of-terrain`?
if so, we need to fix this here: https://github.com/zerocracy/judges-action/blob/7afc6cd80ad0b0827072292644833b71ff4a7b93/judges/dimensions-of-terrain/dimensions-of-terrain.rb#L92
because when the repository is empty, we can get an error, when we call `Fbe.octo.tree`, for example:
https://api.github.com/repos/Yegorov/empty-repo/git/trees/master?recursive=true

Reply:
@Yegorov any repository should be processable, including empty ones

Reply:
@yegor256 ok, but if fetch files for empty repo, we get error:
```
3.3.4 :005 > resp = octo.tree('Yegorov/empty-repo', 'main')                                                                                 
E: GET https://api.github.com/repos/Yegorov/empty-repo/git/trees/main                                                                       
E: User-Agent: "Octokit Ruby Gem 9.1.0"                                                                                                     
   Accept: "application/vnd.github.v3+json"                                                                                                 
   Content-Type: "application/json"                                                                                                         
   Authorization: ********ReSk"                                                                                                             
E: Status 409                                                                                                                               
E: date: "Wed, 09 Oct 2024 11:32:53 GMT"                                                                                                    
   content-type: "application/json; charset=utf-8"                                                                                          
   content-length: "125"                                                                                                                    
   github-authentication-token-expiration: "2025-08-25 00:00:00 +0300"                                                                      
   x-github-media-type: "github.v3; format=json"                                                                                            
   x-accepted-github-permissions: "contents=read"                                                                                           
   x-github-api-version-selected: "2022-11-28"                                                                                              
   x-ratelimit-limit: "5000"                                                                                                                
   x-ratelimit-remaining: "4999"                                                                                                            
   x-ratelimit-reset: "1728477173"                                                                                                          
   x-ratelimit-used: "1"
   x-ratelimit-resource: "core"
   access-control-expose-headers: "ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset"
   access-control-allow-origin: "*" 
   strict-transport-security: "max-age=31536000; includeSubdomains; preload"
   x-frame-options: "deny"
   x-content-type-options: "nosniff"
   x-xss-protection: "0"
   referrer-policy: "origin-when-cross-origin, strict-origin-when-cross-origin"
   content-security-policy: "default-src 'none'"
   vary: "Accept-Encoding, Accept, X-Requested-With"
   server: "github.com"
   x-github-request-id: "B9A0:2487CA:E8CD2D:EB12C1:670669E5"
E: {"message":"Git Repository is empty.","documentation_url":"https://docs.github.com/rest/git/trees#get-a-tree","status":"409"}
D: Octokit::Client.tree("Yegorov/empty-repo", "main") in 457ms
/home/artem/.rvm/gems/ruby-3.3.4/gems/octokit-9.1.0/lib/octokit/response/raise_error.rb:14:in `on_complete': GET https://api.github.com/repos/Yegorov/empty-repo/git/trees/main: 409 - Git Repository is empty. // See: https://docs.github.com/rest/git/trees#get-a-tree (Octokit::Conflict)
```
Should we consider this case or judge should be fail?

Reply:
@Yegorov maybe we can check first, how many files the repo has?

Reply:
@yegor256 Yes, an empty repository can be checked for `size` attribute
```
3.3.4 :023 > octo.repo('Yegorov/empty-repo')                                                                                                
D: Octokit::Client.repo("Yegorov/empty-repo") in 5ms                                                                                        
 =>                                                                                                                                         
{:id=>841899997,
 :node_id=>"R_kgDOMi5f3Q",
 :name=>"empty-repo",
 :full_name=>"Yegorov/empty-repo",
 :private=>false,
 :owner=> ...
 ...
 :size=>0,
```
This discussion has an indirect attitude to this PR. 
@Suban05 Probably, can you create new issue for this. and I can fix it
@yegor256 If everything is ok, please, merge this pr